src/apps/chifra/internal/transactions/validate.go

```diff
@@ -16,10 +16,6 @@ func (opts *TransactionsOptions) validateTransactions() error {
 		return opts.BadFlag
 	}
 
-	if opts.Cache {
```

Comment:
Should we restore this file so that it doesn't allow to use cache? It can be done with `git checkout develop -- src/apps/chifra/internal/transactions/validate.go`

Reply:
Yes. Probably. But you aren't you enabling caching in `chifra transactions` soon?